Doc/library/cgitb.rst

```diff
@@ -42,16 +42,25 @@ analysis.
    .. index:: single: excepthook() (in module sys)
 
    This function causes the :mod:`cgitb` module to take over the interpreter's
-   default handling for exceptions by setting the value of :attr:`sys.excepthook`.
-
-   The optional argument *display* defaults to ``1`` and can be set to ``0`` to
-   suppress sending the traceback to the browser. If the argument *logdir* is
-   present, the traceback reports are written to files.  The value of *logdir*
-   should be a directory where these files will be placed. The optional argument
-   *context* is the number of lines of context to display around the current line
-   of source code in the traceback; this defaults to ``5``. If the optional
-   argument *format* is ``"html"``, the output is formatted as HTML.  Any other
-   value forces plain text output.  The default value is ``"html"``.
+   default handling for exceptions by setting the value of
+   :attr:`sys.excepthook`.
+
+   The optional argument *display* defaults to ``1`` and can be set to ``0``
+   to suppress sending the traceback to the browser. A message single line
+   message declaring that "A problem has occurred ..." will continue to be
+   generated.
+
+   If the argument *logdir* is present, the traceback reports are written to
+   files.  The value of *logdir* should be a directory where these files will
+   be placed.
+
+   The optional argument *context* is the number of lines of context to
+   display around the current line of source code in the traceback; this
+   defaults to ``5``.
```

Comment:
Here I would rather: 'the default value is ``5``. Same thing for the other ones in the file. IMHO, of course.

Reply:
Why?  what’s the difference?

Reply:
Perhaps there is no semantic difference between the two versions but I don't prefer yours. :)

Reply:
sorry what is my version?

Reply:
Your version says "defaults to ``5``."

Reply:
I am not the PR author!

Reply:
Sorry, my mistake! :D I've just noticed that. Sorry, again.

Reply:
Thanks for the feedback, I would like to keep as it is thought as `this defaults to ...` is already used elsewhere in this part of the documentation and I think consistency might be better.